app_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AppController
```

Comment:
Expected `app_controller.rb` to change, as it was modified in 83% of past changes involving `app_template.html`